raspios/overlay-rootfs/etc/systemd/system/cogip-basket.service

```diff
@@ -9,7 +9,7 @@ Type=simple
 Restart=on-failure
 User=root
```

Comment:
`User=root` is the default, you can remove it in all service files.

Reply:
Done in another commit.